Remotion/Web/Core/Res/Themes/NovaViso/HTML/FormGrid.css

```diff
@@ -1,105 +1,84 @@
 /*  Style applied to the form grid table's outer structure  */
-table.formGridTable
+.formGridTable
 {
   border-collapse: collapse;
-  border-width: 0;
   width: 100%;
+  background-color: var(--color-standard-background);
 }
 
-td.formGridTitleCell,
-td.formGridSubTitleCell
+.formGridTitleCell,
+.formGridSubTitleCell
 {
   font-weight: bold;
   font-size: 125%;
```

Comment:
font-weight will be done via font (Roboto-bold).
font-size will be different for title and subtitle. We also want to do a remotion-themed-heading-fontsize and remotion-themed-subheading-fontsize for this. Please check with @derlistigelurch for the names. He's doing the fonts.